docs/tech/glossary.md

```diff
@@ -18,6 +24,13 @@ Byzantine Fault Tolerance { #byzantine_fault_tolerance }
 Canary Network { #canary_network }
 : A network used for testing features under “real fire” conditions, before deploying them on the [main network](#main_network). All users of the canary network are real users, but they are aware of the experimental nature of the platform. The name comes from the time when actual miners used actual canaries to detect the presence of poisonous gas in the mines. Flare's canary network is called [Songbird](#songbird).
 
+Clear-cut decidable { #clear-cut_decidable }
+: Clear-cut decidability requires having a synchronized view on data from external data sources (e.g. other chains) that are used for data attestations.
+For example, in slower block-producing blockchains like Bitcoin, one [attestation provider](#attestation-provider) may see a certain block at the moment of query while another one may not see it yet,
+since the block might not have been fully distributed throughout the network.
+Such providers would yield completely different attestations.
+Therefore, special data view synchronization protocols have to be applied.
```

Comment:
I'm not a fan of such short sentences. How about "Such providers would yield..., therefore special data..." ?

Reply:
Fixed in https://github.com/flare-foundation/docs/pull/14/commits/1b5351ffe61bae09084cc51cf625f8a5da497c8f